actions/complete/index.js

```diff
@@ -9,15 +9,52 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
+
+const openwhisk = require('openwhisk');
 const { asGenericAction } = require('../GenericAction.js');
 const { asAuthAction } = require('../AuthAction.js');
-const { asFirefallAction } = require('../FirefallAction.js');
+
+const STATUS_RUNNING = 'running';
+const STATUS_COMPLETED = 'completed';
 
 async function main(params) {
-  const {
-    prompt, temperature, model, firefallClient,
-  } = params;
-  return firefallClient.completion(prompt ?? 'Who are you?', temperature ?? 0.0, model ?? 'gpt-4');
+  const { jobId } = params;
+
+  const ow = openwhisk();
+
+  if (!jobId) {
```

Comment:
Do you think this logic might be helpful for other actions as well? Should we move it outside of action to make it reusable?

Reply:
It seems this is needed for this action only.